src/Altinn.Dan.Plugin.Arbeidstilsynet/Utils/BRUtils.cs

```diff
@@ -0,0 +1,72 @@
+using Altinn.Dan.Plugin.Arbeidstilsynet.Models;
+using Altinn.Dan.Plugin.Arbeidstilsynet.Models.Sub;
+using Nadobe.Common.Exceptions;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+
+namespace Altinn.Dan.Plugin.Arbeidstilsynet.Utils
+{
+    public class BRUtils
+    {
+        public static async Task<BREntityRegisterEntry> GetMainUnit(string organizationNumber, HttpClient client)
+        {
+            var org = await GetOrganizationInfoFromBR(organizationNumber, client);
+
+            if (org != null && !string.IsNullOrEmpty(org.OverordnetEnhet))
```

Comment:
I know we disagree about this, but: https://programmerbyday.wordpress.com/2020/11/23/why-you-should-always-use-curly-braces-for-single-statement-blocks-in-csharp/

Reply:
Agree to disagree :)

Reply:
@ctrlaltd3v It's a well known fact that @erlendoksvoll is wrong on this subject :)

Reply:
I'm with @erlendoksvoll on this one. I've always felt the classical "arguments" against dropping braces presented in that blog (and million others parroting the same) are utterly contrived and never something that actually occurs in the real world, while curly braces around the most trivial `if`s just add visual noise and reduces the amount of actual code you can fit in a screenful ultimately causing more scrolling. And if you don't manage your indentations right the code gets unreadable regardless of braces.

Reply:
Hear, hear! Two against one 😂